src/partials/head-prelude.hbs

```diff
@@ -2,7 +2,11 @@
     <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=5, minimum-scale=1">
     <meta name="msvalidate.01" content="B5163518CBE4A854B63801277FE5E35C" />
     <meta name="google-site-verification" content="ucqagxjVuq0lJZeLKs0F5AppzK111lNt3IoxU6mzlJE" />
+    {{#if (page.attributes.no-index) }}
```

Comment:
I think we should invert the check and use `{{#if (ne page.attributes.no-index undefined) }}` instead because an attribute is a string value, so if we declare `:page-no-index:`, `page.attributes.no-index` will equals `''` (empty string) and I believe that empty string is false.

@recrwplay What do you think?